resharper/resharper-unity/src/Rider/UnityYamlOccurrenceInformationProvider.cs

```diff
@@ -0,0 +1,50 @@
+using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Feature.Services.Occurrences;
+using JetBrains.ReSharper.Feature.Services.Occurrences.OccurrenceInformation;
+using JetBrains.ReSharper.Plugins.Unity.Yaml.Feature.Services.Occurrences;
+using JetBrains.ReSharper.Psi;
+using JetBrains.Util;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Rider
+{
+    [SolutionFeaturePart]
+    public class UnityYamlOccurrenceInformationProvider : IOccurrenceInformationProvider
```

Comment:
What does this class do? `UnityEditorOccurrence` is a `ReferenceOccurrence`, so does that handle everything for us already?

Reply:
Thanks! It is out-of-date class, befoure my UnityEditorOccurence was not child of ReferenceOccurrence. I will remove it